ui.tests/test-module/libs/commons/localeDataSets.js

```diff
@@ -1,6 +1,6 @@
 const selectors = {
     ACCORDION: '.cmp-accordion > label',
-    BUTTON: '.button.base:not(.submit):not(.reset) span',
+    BUTTON: 'span.cmp-adaptiveform-button__text',
```

Comment:
Why do you need span here, ideally if you are unable to use via BEM, we need to add BEM to it too. Please check and fix this same. For example, even the CSS selector of check_box_item1 looks incorrect.